scss/main.scss

```diff
@@ -52,8 +52,14 @@ h1,h2,h3 {
 }
 
 hr {
-  background-color: #BABABA;
-  margin: 0;
+  background-color:$light-grey;
+  margin: 0 6rem;
```

Comment:
It's hard to tell from the figma but it looks like only the middle hr has the margin. The header and footer ones take up the whole width of the page (like you originally had it). Then again it also looks like the hr disappears above 1440px so maybe ask the designers what they think?

Reply:
Ok, I made changes so that only the inner hr has margins and disappears after 1440px. I'll put it in design review to get feedback.

Reply:
Joseph has confirmed this is the correct design.